src/localize/languages/sv.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "../schema/schema.json",
+  "common": {
+    "name": "Meteoalarm Card",
+    "description": "Meteoalarm card varnar dig om aktuella väderhändelser.",
+    "warning": "Varning",
+    "expected": "förväntad",
+    "unavailable": {
+      "long": "Integrationen är inte tillgänglig",
+      "short": "Inte tillgänglig"
+    }
+  },
+  "error": {
+    "missing_entity": "Det är nödvändigt att ange en enhet!",
+    "invalid_integration": "Denna integration är inte giltig!",
+    "entity_invalid": {
+      "single": "Den valda integrationen matchar inte den valda enheten.",
+      "multiple": "Den valda integrationen matchar inte de valda enheterna: {entity}."
+    }
+  },
+  "editor": {
+    "entity": "Enhet",
+    "integration": "Integration",
+    "required": "Krävs",
+    "recommended": "Rekommenderas",
+    "override_headline": "Överrida titel",
+    "hide_when_no_warning": "Göm när det inte finns några varningar",
+    "hide_caption": "Dölj bildtext",
+    "disable_swiper": null,
```

Comment:
What's up with that?

Reply:
Yeah I wasn't sure how to translate disable_swiper so i left it untranslated.